zerver/openapi/zulip.yaml

```diff
@@ -17668,6 +17668,14 @@ paths:
             Title to use for the BigBlueButton meeting.
 
             A good choice is something like "{stream_name} meeting".
+        - in: query
+          name: voice_only
+          schema:
+            type: boolean
+          required: true
+          description: |
+            Whether the call should be audio-only, in which case we keep
+            cameras disabled for this call.
```

Comment:
This needs `Changes: New in Zulip 8.0 (feature level xyz).`. See, for example, any other PR that updates `API_FEATURE_LEVEL`.

Reply:
It's my 1st time making an API change, so this really helped, thank you! I was initially a little lost coz I was looking at the wrong `changelog.md` (the one in `docs/overview`)

Have updated `api-docs/changelog.md`